### ISSUE TITLE
Failure during startup when no kubenetes context is set

### ISSUE THREAD

**Issue by OmarHawk** [rancher-sandbox/rancher-desktop#2551]
### Actual Behavior

After initial installation we had the case that no kubernetes context was set within the application. During startup, the application failed then to properly initialize, with such an log output:

2022-07-13T09:00:46.081Z: WSL: executing: cat /root/.docker/config.json: Error: wsl.exe exited with code 1

and further output (don't have that anymore :-() that really indicated that trying to set the kubenetes context with kubectl run by Rancher Desktop was the reason for the failure. So this list:

![grafik](https://user-images.githubusercontent.com/2911492/179171464-43abd280-e240-4ecd-8fff-10d744299f7e.png)

 had none selected and only after selecting one entry there and then restarting the application it started to work.

### Steps to Reproduce

Sorry for being lazy, but it is probably:

- Make sure to clear kubernetes context
- Start the application

### Result

Application not initialized and non-functional.

### Expected Behavior

Application starting up properly

### Rancher Desktop Version

1.4.1

### Rancher Desktop K8s Version

1.23.3

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 Version 21H2

### What CPU architecture are you using?

x64

### Windows User Only

N/A

**Comment by jandubois** [comment]
This was filed against RD 1.4.1 on Windows with no logs and no follow-up. The WSL backend startup logic has been substantially rewritten since then, and the current code correctly handles startup when no Kubernetes context is selected. If you still see a startup failure on a current release, please open a new issue with the log bundle from the Troubleshooting page — that context is essential for diagnosis.
